system_baseline/views/v1.py

```diff
@@ -698,16 +698,16 @@ def delete_systems_with_baseline(baseline_id, system_ids):
     current_app.logger.audit(message, request=request, success=True)
 
     system_ids = baseline.mapped_system_ids()
-    return system_ids
+    return "OK"
 
 
 def create_deletion_request_for_systems(baseline_id, system_ids):
     ensure_rbac_write()
     validate_uuids([baseline_id])
     validate_uuids(system_ids)
 
-    return_value = delete_systems_with_baseline(baseline_id, system_ids)
-    return return_value
+    delete_systems_with_baseline(baseline_id, system_ids)
+    return "OK"
```

Comment:
Actually, I approved, but when I was looking to see the best order to merge to prevent conflicts, I noticed this spot is shared in both and is not the same:
https://github.com/RedHatInsights/system-baseline-backend/pull/164/files#diff-285e0634d3b94393071de1f685907e13d80487fa2e250ae076868664473fbad3R711-R712